definitions/v9/unionfansub.yml

```diff
@@ -98,8 +98,27 @@ search:
       filters:
         - name: querystring
           args: cat
-    title:
+    fansub:
+      selector: td:nth-child(2) .fansubbd
+    title_default:
       selector: td:nth-child(2) a
+    title_season:
+      text: "{{ .Result.title_default }}"
+      filters:
+        - name: re_replace
+          args: ["(?i)[ºª]", ""]
```

Comment:
Do this only when proceeded by 1-4 numerical characters?

Reply:
There really can be any number, in Spanish º and ª are used to indicate any cardinality.

Or by 1-4 do you mean the amount of characters?

Reply:
The number of characters, so everything from `1` to `9999`. Unless those characters would ever appear after a year, in which case just 1-3 numerical characters (`1` to `999`).